async-query-core/src/main/java/org/opensearch/sql/spark/validator/SQLQueryValidationVisitor.java

```diff
@@ -561,25 +561,29 @@ public Void visitFunctionName(FunctionNameContext ctx) {
   }
 
   private void validateFunctionAllowed(String function) {
-    FunctionType type = FunctionType.fromFunctionName(function.toLowerCase());
+    String functionLower = function.toLowerCase();
```

Comment:
what is functionLower? function name in lower case? if so, could we have better naming?